lib/mayaUsd/CMakeLists.txt

```diff
@@ -259,6 +255,7 @@ target_link_libraries(${PROJECT_NAME}
         ${UFE_LIBRARY}
         ${MAYA_LIBRARIES}
         usdUfe
+        $<$<BOOL:$<VERSION_GREATER_EQUAL:${USD_VERSION},0.24.11>>:Boost::headers>
```

Comment:
Since USD no longer has Boost, we need to add a Boost dependency so cmake will add the boost include path (for the Boost headers we still use).